### ISSUE TITLE
v101.0.4951.39 not on Bromite F-Droid repo

### ISSUE THREAD

**Issue by coderkei** [bromite/bromite#2035]
### Preliminary checklist

- [X] I have read the [README](https://github.com/bromite/bromite/blob/master/README.md).
- [X] I have searched the existing issues for my problem. This is a new ticket, NOT a duplicate or related to another open issue.
- [X] I have read the [FAQs](https://github.com/bromite/bromite/blob/master/FAQ.md).
- [X] I have updated Bromite to the latest version. The bug is reproducible on this latest version.

### Can the bug be reproduced with corresponding Chromium version?

No

### Bromite version

101.0.4951.39

### Device architecture

arm64-v8a

### Android version

11

### Device model

Oppo Reno

### Changed flags

no flags changes

### Is this bug about the SystemWebView?

No

### Is this bug happening in an incognito tab?

No

### Is this bug caused by the adblocker?

No

### Is this bug a crash?

No

### Describe the bug

Bromite's F-Droid repo appears to not have the version 101.0.4951.39 some days after it was released and Bromite now shows the update nag icon on the 3 dot menu.

I am unable to update to this version as Bromite's F-Droid repo is still showing the latest version for arm64-v8a is 100.0.4896.135

### Steps to reproduce the bug

1) Install official F-Droid app.
2) Add the Bromite Repo following the instructions [located here.](https://www.bromite.org/fdroid) 
3) Refresh repositories
4) F-Droid only shows the older version being available.

### Expected behavior

101.0.4951.39 to be available on the repo as an update.

### Screenshots

![Screenshot_2022-05-01-20-27-38-69_66a3b1c2c181ff92b8bf945e748b6fb7](https://user-images.githubusercontent.com/7037851/166161637-c39184d0-a12c-4bf0-8595-8e2805293788.jpg)

**Comment by pitsi** [comment]
I noticed it too, since the day v101.x became available.

I use fdroidcl to install and update bromite and its repo the only fdroid repo I have. The `fdroidcl update` command, which is for updating the index of the repo says that it was not modified
```
$ fdroidcl update
Downloading https://fdroid.bromite.org/fdroid/repo/index-v1.jar... not modified
```
and the latest version it shows as available is 100.0.4869.135
```
$ fdroidcl show org.bromite.bromite 
Package          : org.bromite.bromite
Name             : Bromite
Summary          : Bromite is Chromium plus ad blocking and enhanced privacy; take back your browser
Added            : 2018-07-14
Last Updated     : 2022-04-16
Version          :  (1650070930)
License          : BSD-3-Clause
Categories       : Internet
Website          : https://www.bromite.org
Source Code      : https://github.com/bromite/bromite
Issue Tracker    : https://github.com/bromite/bromite/issues
Changelog        : https://github.com/bromite/bromite/blob/master/CHANGELOG.md
Donate           : https://www.bromite.org/#donate
Bitcoin          : bitcoin:bc1qlx7h0lj9z88g2xfeuwsrtfs77sxuhrxf2t28sw

Description :


Available Versions :

    Version : 100.0.4896.135 (1650070930)
    Size    : 168955241
    MinSdk  : 0
    ABIs    : x86_64

    Version : 100.0.4896.135 (1650070920)
    Size    : 157724085
    MinSdk  : 0
    ABIs    : arm64-v8a

    Version : 100.0.4896.135 (1650070880)
    Size    : 156245285
    MinSdk  : 0
    ABIs    : x86

    Version : 100.0.4896.135 (1650070870)
    Size    : 118116607
    MinSdk  : 0
    ABIs    : armeabi-v7a
```
There is even a date for the last time the package was updated, and it obviously refers to the forementioned release. Likewise, the remaining packages for bromite webview, chromium and chromium webview are also in that previous version.

**Comment by csagan5** [comment]
It will be fixed on next release.
